packages/uui-card-content-node/lib/uui-card-content-node.element.ts

```diff
@@ -113,21 +113,37 @@ export class UUICardContentNodeElement extends UUICardElement {
     return html`<uui-icon .svg="${this.fallbackIcon}"></uui-icon>`;
   }
 
+  // This is deprecated - use href instead
+  #renderDeprecatedButton() {
```

Comment:
I don't think we should see it as Deprecated. Its still a valid feature use-case to not add href and use a click event instead. But yes, In New Backoffice we would like to have links behind all cards if possible.